src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -320,8 +322,10 @@ public virtual void Close()
             finally
             {
                 this.initialized = false;
-
-                EqtTrace.Warning("ProxyOperationManager: Timed out waiting for test host to exit. Will terminate process.");
+                if (testHostExitedWithinTimeout == false)
```

Comment:
```suggestion
                if (!testHostExitedWithinTimeout)
```

but I like it if it's done on purpose so feel free to keep as-is